app/src/main/java/com/therealbluepandabear/pixapencil/activities/canvas/CanvasActivity+onBackPressed.kt

```diff
@@ -17,9 +15,9 @@ fun CanvasActivity.extendedOnBackPressed() {
             getString(R.string.dialog_unsaved_changes_message_in_code_str),
             getString(R.string.dialog_positive_button_text_in_code_str),
             { _, _ ->
-                startActivity(Intent(this, MainActivity::class.java))
+                finish()
             },  getString(R.string.dialog_negative_button_text_in_code_str), { _, _ -> })
     } else {
-        startActivity(Intent(this, MainActivity::class.java))
+        finish()
```

Comment:
Create a function which encapsulates this to avoid cope duplication.

Reply:
I think moving this one line of code to a function don't give any benefit. If you create a new function for this one line you also have to call this new function in 2 place

Reply:
Fair enough.